pages/dkp/kommander/2.2/projects/project-policies/index.md

```diff
@@ -10,7 +10,7 @@ Project Role Bindings grant access to a specified Project Role for a specified g
 
 ## Configure Project Role Bindings - UI Method
 
-Before you can create a Project Role Binding, ensure you have created a Group. A Kommander Group can contain one or several Identity Provider users and or groups.
+Before you can create a Project Role Binding, ensure you have created a Group. A Kommander Group can contain one or several Identity Provider users and/or groups.
```

Comment:
```suggestion
Before you can create a Project Role Binding, ensure you have created a Group. A Kommander Group can contain one or several Identity Provider users or groups.
```

Reply:
I'm don't think this suggested change is accurate. You can add Identity Provider users and/or groups to a Kommander group. It's not one or the other. I think we should change it back.

Reply:
It's up to you.. I committed based on thinking Natalie as an Engineer knew how it worked, but I am ok with having and/or.  You decide and I'll fix it.  ALso Craig Neth helped me back that other goop out of this PR.  SOrry about that!

Reply:
My suggestion was purely a grammatical fix and to avoid merging a vale error. When I read the second sentence to me it seemed that it _or_ suggested that you can add either one or several Identity Providers or groups. Saying either of those is an option and not an exclusive option. Feel free to change this to whatever makes sense for the sentence structure.

Reply:
I've spoken to Natalie and she had googled what to replace `and/or` with and the internet said to replace it with `or`. The reason she did this is because of the Vale error saying you shouldn't use `and/or`. Do you have a recommendations on the best way to word this to get around the vale error but still accurately explain what's possible with the form?

Reply:
Got it!  Will do!

Reply:
Wanted to chime in:

1. and/or is the appropriate usage within context.
2. Vale errors are suggestions, you can choose to ignore and merge if you get error messages, Vale gets stuff wrong.